bundletester/spec.py

```diff
@@ -311,6 +311,8 @@ def filter_yamls(yamls):
 
     result = []
     for yamlfn in yamls:
+        if yamlfn.endswith('metrics.yaml'):
+            continue
```

Comment:
An out and out match would make me feel a little bit better. Do people name their metrics "foo-metrics.yaml"?